package exceptionhandle.exceptionhandle;

public class Animal {
	private int legs = 4;
	static void walk() {
		System.out.println("Animal is walking");
	}
	private void aa() {  
		System.out.println("Just to check that private method can access or not");
	}

}
